packages/zero-client/src/client/zero.ts

```diff
@@ -524,6 +529,13 @@ export class Zero<
     );
     this.#clientToServer = clientToServer(schema.tables);
 
+    this.#deleteClientsManager = new DeleteClientsManager(
+      () => this.#connectResolver.promise,
+      msg => this.#send(msg),
+      rep.memdag,
```

Comment:
I think this should be the perdag.  

When working with the CLIENT_GROUPS_HEAD_NAME and CLIENTS_HEAD_NAME heads, replicache works directly with the perdag.

If we don't write this to the perdag, nothing currently will persist it to the perdag.  Persist only operates on the Commit chain, it doesn't persist all memdag heads to the perdag.